constrain/library/G36OutdoorAirDamperPositionForReturnFanDirectBuildingPressure.py

```diff
@@ -1,34 +1,63 @@
 """
-G36 2021
-
 ### Description
 
+This verification aims to check if the outdoor air damper operates correctly in systems with return fan direct building pressure control. The damper should maintain a position that supports proper building pressurization control.
+
+### Code requirement
+
+- Code Name: ASHRAE Guideline 36
+- Code Year: 2021
+- Code Section: 5.16.2 Supply Air Temperature Control
+- Code Subsection: 5.16.2.3 Outdoor Air Damper Control with Return Fan Direct Building Pressure
+
+### Verification Approach
+
 Section 5.16.2.3
+- Supply air temperature shall be controlled to setpoint using a control loop whose output is mapped to sequence the heating coil (if applicable), outdoor air damper, return air damper, and cooling coil.
```

Comment:
?

Reply:
Based on our group discussion, we've decided to leave it as is.